code/game/objects/items/holy_weapons.dm

```diff
@@ -307,7 +307,7 @@
 	slot_flags = ITEM_SLOT_BELT
 
 /obj/item/nullrod/claymore/katana
-	name = "hanzo steel"
+	name = "Hanzo steel"
```

Comment:
Any reason why X-ray remained x-ray and this gets a capitalisation? I don't know how the game parses names.

Reply:
It's a kill bill reference, a guy named Hattori Hanzo made the best katanas that the Bride seeks out, so that she can kill bill. Characters in the movie refer to her katana as "Hanzo steel".

Reply:
It's a name. X-ray is a name. Both of these are capitalised normally. The changes leave it as x-ray and X-ray.

Reply:
Corrected X-ray.